filebeat/crawler/prospector_log.go

```diff
@@ -265,19 +265,20 @@ func (p *ProspectorLog) continueExistingFile(h *harvester.Harvester, previousFil
 func (p *ProspectorLog) resumeHarvesting(h *harvester.Harvester, offset int64) {
 
 	logp.Debug("prospector", "Start / resuming harvester of file: %s", h.Path)
-	h.SetOffset(offset)
-	h.Start()
+	newHarvester, _ := p.Prospector.AddHarvester(h.Path, h.Stat)
+	newHarvester.SetOffset(offset)
+	newHarvester.Start()
```

Comment:
@urso I removed this code again as it is not needed. I was playing around with creating a new harvester every time offset changes etc. And this is a left over. Potentially we need something like harvester copy.